pyteal/ast/abi/tuple.py

```diff
@@ -198,9 +198,6 @@ def _index_tuple(
         if offset == 0:
```

Comment:
combine?

Reply:
yes, doable, need to rephrase comments a bit.

Reply:
actually, by this discussion thread: https://github.com/algorand/pyteal/pull/663#discussion_r1102015352, I am pushing back a bit.